src/components/WheelSelect/wheelPicker.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+
+import ApplyStyle from 'components/WheelSelect/applyStyle';
+
+class WheelPicker extends React.Component {
```

Comment:
@rafayet-monon Why don't we use `https://kenwheeler.github.io/slick/` what we have instead of create a new one?

Reply:
Use slick in [8298612](https://github.com/rafayet-monon/surveyor/pull/19/commits/82986129767ac03209279fbac846a4200d726325)